### DISCUSSION TITLE
Purchase the license but unable to download full video, it download the short length videos

### DISCUSSION THREAD

**Discussion by ghost** [aclap-dev/video-downloadhelper#1490] [Help and Q&A]
### Your question

Today i have purchase the license on my macbook for chrome browser, i have applied the licnense reload the extension but it won't download the full length video. Download short video and the file size also not more than 5mb, Please provide the solution, fix the issue or refund me. Thanks

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"3e29775dd8ab4af7"}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 3
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Your problem description is too vague to allow anybody else to investigate it.  You need to supply some URLs of web pages on which there are videos that are giving you trouble.  Specific videos.  Not names of web sites.  URLs of web pages that host videos.
